pyServer/GuestFS.py

```diff
@@ -98,9 +98,10 @@ def start(self):
         self.rootLogger.info('GuestFish:Create:Local> ' + ' '.join(redactedArgs))
 
         try:
-            output = subprocess.check_output(args, env=self.environment, universal_newlines=True)
+            output = subprocess.check_output(args, env=self.environment, universal_newlines=True, stderr = subprocess.STDOUT)
```

Comment:
In Python, does the ordering of these parameters matter? In Python doc, it looks like 'stderr' precedes env and universal_newlines: 

https://docs.python.org/2/library/subprocess.html#subprocess.Popen

Reply:
ordering doesn't affect as we are providing variable name. But anyway removing it as redirection of stdout/stderr can't work as expected for our libguestfs